docker/ruby/Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN apt-get update && apt-get install -y \
   locales \
   vim \
   git \
-  netcat
+  netcat \
+  postgresql-client
```

Comment:
This will install the `postgres-client` , so adding the optional install and the build `ARG` is unnecessary. Let's choose one or the other.

Reply:
I removed this so that we rely on the arg to decide if we should install in the container or not.